### PR TITLE
Added a privacy policy page

### PR DESCRIPTION
**Pull request by abdulr07**

### CODE REVIEW

**Comment by chrisgumieny** [comment]
Is this deployed anywhere?

**Comment by abdulr07** [comment]
It is not deployed yet, Viona will deploy it in about an hour

File: src/App.js [hunk 1]
```diff
@@ -14,7 +14,7 @@ import { ThirdPartyEducation } from "./components/ThirdPartyEducation"
 import { Search } from "./components/Search"
 import Help from './components/Help'
 import ProfileChange from "./components/ProfileChange"
-
+import Policy from "./components/Policy"
```
Comment (vionaveseli) [inline src/App.js]: import {Policy} from "./components/Policy"

File: src/App.js [hunk 2]
```diff
@@ -36,6 +36,7 @@ function App() {
               <Route path="/search"> < Search /> </Route>
               <Route path="/help"> <Help /> </Route>
               <Route path="/"> <Home /> </Route>
+              <Route path="policy"> <Policy /> </Route>
```
Comment (vionaveseli) [inline src/App.js]: <Route path="/policy"> <Policy /> </Route>

**Review by vionaveseli — COMMENTED** [review]
Fix these two lines for App.js then you're good to go! (:

File: src/App.js (hunk 2 above)
Comment (vionaveseli) [reply src/App.js]: <Route path="/policy"> <Policy /> </Route>

**Comment by vionaveseli** [comment]
https://auth-development-751fc.web.app/policy 

deployed page with changes made. (Two line changes in App.js)
(I added policy in nav under profile by help)

File: src/components/Policy.js [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+                        Your Rights and Choices
+                        </Accordion.Toggle>
+                    </Card.Header>
+                        <Accordion.Collapse eventKey="3">
+                            <Card.Body>You have the right to request that Crypto Pro disclose certain information to you about its collection and use of your personal information. After Crypto Pro receives and confirms your verifiable request, it will disclose to you: •	The categories of personal information we collected about you.
+-The categories of sources for the personal information we collected about you.
+-Our business or commercial purpose for collecting that personal information.
+-The categories of third parties with whom we share that personal information.
+-The specific pieces of personal information we collected about you.
+</Card.Body>
```
Comment (IKwiotek) [inline src/components/Policy.js]: This section is not displaying as a list on the hosted site. Is this expected to be displayed as a list?

File: src/components/Policy.js [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+                        Crypto Pro may use, or disclose the personal information it collects for one or more of the following business purposes:
+                        </Accordion.Toggle>
+                    </Card.Header>
+                        <Accordion.Collapse eventKey="0">
+                            <Card.Body> -To fulfill or meet the reason you provided the information. For example, to provide services.
+-To help maintain the safety, security, and integrity of Crypto Pro’s site and services. For example, Crypto Pro may screen and use the information to protect against spammers, hackers, and others who pose harm to the site and services; Crypto Pro may share information with moderation service providers; and Crypto Pro may share the information with law enforcement.
+-To evaluate and improve the site and services. For example, Crypto Pro may aggregate and/or de-identify information to produce statistical data.
+-To monitor and enforce user compliance with the guidelines of the law and its policies. 
+-To respond to law enforcement requests and as required by applicable law, court order, or governmental regulations.
+</Card.Body>
```
Comment (IKwiotek) [inline src/components/Policy.js]: This section is not displaying as a list on the hosted site. Is this expected to be displayed as a list?

**Review by IKwiotek — CHANGES_REQUESTED** [review]

File: src/components/Policy.js (hunk 3 above)
Comment (abdulr07) [reply src/components/Policy.js]: It doesn't have to be displayed as a list, I'll adjust it and make it into paragraph format

**Comment by vionaveseli** [comment]
new deployment:  https://auth-development-751fc.web.app/policy

**Review by vionaveseli — APPROVED** [review]
changes have been deployed. Everything looks great! good job

**Review by chrisgumieny — APPROVED** [review]
Everything looks good!

**Review by IKwiotek — APPROVED** [review]
It still seems like some of these sections should be a list, but I'll approve as we are attempting to meet the project deadline.

**Pull request merged**